source/borderrouter_tasklet.c

```diff
@@ -233,6 +236,14 @@ static void load_config(void)
     br.ra_life_time = cfg_int(global_config, "RA_ROUTER_LIFETIME", 1024);
     br.beacon_protocol_id = cfg_int(global_config, "BEACON_PROTOCOL_ID", 4);
 
+    rf_prefix_from_backhaul = cfg_int(global_config, "PREFIX_FROM_BACKHAUL", 0);
+
+    /*If rf_prefix_from_backhaul is set to true, then 6lp interface should use
+     * the same prefix as backhaul*/
+    if (rf_prefix_from_backhaul) {
```

Comment:
Are you considering the `NET_IPV6_BOOTSTRAP_AUTONOMOUS` mode at all?

If that option is in action, then backhaul prefix is learned from Ethernet, and then used on 6LoWPAN. Therefore 6LoWPAN cannot be started before the backhaul if that mode is enabled.